### ISSUE TITLE
tracing-journald feature fails the build

### ISSUE THREAD

**Issue by Fuuzetsu** [asayers/tailsrv#5]
On current master

```shell
[nix-develop]$ cargo check --features tracing-journald
    Checking tailsrv v0.9.0 (/home/shana/programming/tailsrv)
error[E0609]: no field `journald` on type `fn() -> impl Parser<Opts> {opts}`
   --> src/main.rs:443:13
    |
443 |     if opts.journald {
    |             ^^^^^^^^ unknown field

error[E0277]: the `?` operator can only be used in a function that returns `Result` or `Option` (or another type that implements `FromResidual`)
   --> src/main.rs:444:67
    |
433 | fn log_init(#[cfg(feature = "tracing-journald")] journald: bool) {
    | ---------------------------------------------------------------- this function should return `Result` or `Option` to accept `?`
...
444 |         let subscriber = subscriber.with(tracing_journald::layer()?);
    |                                                                   ^ cannot use the `?` operator in a function that returns `()`
    |
    = help: the trait `FromResidual<std::result::Result<Infallible, std::io::Error>>` is not implemented for `()`
help: consider adding return type
    |
433 ~ fn log_init(#[cfg(feature = "tracing-journald")] journald: bool) -> Result<(), Box<dyn std::error::Error>> {
434 |     let subscriber = tracing_subscriber::registry();
...
450 |     subscriber.init();
451 +     Ok(())
    |

Some errors have detailed explanations: E0277, E0609.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `tailsrv` (bin "tailsrv") due to 2 previous errors
```

**Comment by asayers** [comment]
Thanks!  Fixed in 0dd91ce.

**Comment by asayers** [comment]
Released as 0.9.1
